Build failure: opencpn

### Nixpkgs version

- Unstable (25.11)

### Steps to reproduce

- Add opencpn (version 5.10.2 from unstable) to config.
- config will fail to build because it can't import `lzma.h`

### Can Hydra reproduce this build failure?

No, Hydra cannot reproduce this build failure.

### Link to Hydra build job

_No response_

### Relevant log output

```console
> cd /build/source/build/model && /nix/store/bcw9f6r9v2fm3kv7d15fcrya0mf34xds-gcc-wrapper-14.3.0/bin/g++ -DHAVE_LIBUSB_10 -DHAVE_WEBVIEW -DHAVE_WX_GESTURE_EVENTS -DINTERFACE -DIXWEBSOCKET_USE_OPEN_SSL -DIXWEBSOCKET_USE_TLS -DIXWEBSOCKET_USE_ZLIB -DRAPIDJSON_HAS_STDSTRING=1 -DWXUSINGDLL -D_FILE_OFFSET_BITS=64 -D_OCPN_MODEL -D__OCPN_USE_GLEW__ -D__WXGTK3__ -D__WXGTK__ -DocpnUSE_GL -DocpnUSE_GLSL -DocpnUSE_SVG -DocpnUSE_wxBitmapBundle -DwxUSE_WEBVIEW=1 -I/build/source/build/include -I/build/source/model/include -I/build/source/include -I/nix/store/k29ns0ck4991vassvlzjixfjv228i4yv-glib-2.84.3-dev/include/glib-2.0 -I/nix/store/9151j4k3hwgj35bdz7va4bmlvgpa4pdm-glib-2.84.3/lib/glib-2.0/include -I/nix/store/b5ykcrjx79arrscvdn4a8268b4kr2z0q-libusb-1.0.29-dev/include/libusb-1.0 -I/nix/store/2k8m4nvv5nf3plgbrp8c598jzqp19kk3-gtk+3-3.24.49-dev/include/gtk-3.0 -I/nix/store/78blng4qixpb07pmcv4mfrdhj3km0al8-at-spi2-core-2.56.2-dev/include/atk-1.0 -I/nix/store/yqmrid9yml96sqw9ap4m6vygbhnq6ygp-cairo-1.18.4-dev/include/cairo -I/nix/store/r9mw6vr5lf3x0yn31vb3zvsdb4mirlbs-freetype-2.13.3-dev/include/freetype2 -I/nix/store/6pm3l84z718wpigkifam78fp9rnlx3kh-gdk-pixbuf-2.42.12-dev/include/gdk-pixbuf-2.0 -I/nix/store/kdzvxhprkpxpzlbwwgy3c8k9lm91r50a-pango-1.56.3-dev/include/pango-1.0 -I/nix/store/bk38d0r3qy65q61w86b7laihwf5cy2n9-harfbuzz-11.2.1-dev/include/harfbuzz -I/build/source/libs/observable/include -I/build/source/libs/IXWebSocket -I/build/source/libs/garmin/jeeps -I/build/source/libs/gdal/include -I/build/source/libs/geoprim/src -I/build/source/libs/iso8211/include -I/build/source/libs/mDNS/include -I/build/source/libs/mongoose/include -I/build/source/libs/N2KParser/include -I/build/source/libs/nmea0183/src -I/build/source/libs/pugixml -I/build/source/libs/s52plib/src -I/build/source/libs/sound/include -I/build/source/libs/wxJSON/include -I/build/source/libs/std_filesystem/include -I/build/source/libs/wxservdisc -I/build/source/libs/picosha2 -I/build/source/libs/serial/include -isystem /nix/store/py03hrzr0y2nz0qhk2a2av86lab5jc9x-wxwidgets-3.2.8.1/lib/wx/include/gtk3-unicode-3.2 -isystem /nix/store/py03hrzr0y2nz0qhk2a2av86lab5jc9x-wxwidgets-3.2.8.1/include/wx-3.2 -pthread -D__WXGTK3__ -O3 -DNDEBUG -std=gnu++17 -fPIC    -DPREFIX=\"/nix/store/qrbad2z590mw7lf4massbqg1q16mhir5-opencpn-5.10.2\" -Wall -Wno-unused -fexceptions -rdynamic -fno-strict-aliasing -Wno-deprecated-declarations -MD -MT model/CMakeFiles/_model_src.dir/src/comm_ais.cpp.o -MF CMakeFiles/_model_src.dir/src/comm_ais.cpp.o.d -o CMakeFiles/_model_src.dir/src/comm_ais.cpp.o -c /build/source/model/src/comm_ais.cpp
       > In file included from /build/source/model/src/chartdata_input_stream.cpp:40:
       > /build/source/model/include/model/chartdata_input_stream.h:32:10: fatal error: lzma.h: No such file or directory
       >    32 | #include <lzma.h>
       >       |          ^~~~~~~~
       > compilation terminated.
```

### Additional context

First issue here, if I made a mistake, let me know! I was unsure how to navigate Hydra, so to answer that section I used the job status from [opencpn/default.nix](https://github.com/NixOS/nixpkgs/blob/master/pkgs/applications/misc/opencpn/default.nix)

### System metadata

 - system: `"x86_64-linux"`
 - host os: `Linux 6.16.4-xanmod1, NixOS, 25.11 (Xantusia), 25.11.20250905.8eb28ad`
 - multi-user?: `yes`
 - sandbox: `yes`
 - version: `nix-env (Nix) 2.28.5`
 - nixpkgs: `/nix/store/6wpzhvnb026lab4b2pzph2qlql4zwasa-source`

### Notify maintainers

@kragniz 
@lovesegfault 
---

**Note for maintainers:** Please tag this issue in your pull request description. (i.e. `Resolves #ISSUE`.)


### I assert that this issue is relevant for Nixpkgs

- [x] I assert that this is a bug and not a support request.
- [x] I assert that this is not a [duplicate of an existing issue](https://github.com/NixOS/nixpkgs/issues?q=is%3Aopen+is%3Aissue+label%3A%220.kind%3A+build+failure%22). 
- [x] I assert that I have read the [NixOS Code of Conduct](https://github.com/NixOS/.github/blob/master/CODE_OF_CONDUCT.md) and agree to abide by it.

### Is this issue important to you?

Add a :+1: [reaction] to [issues you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[issues you find important]: https://github.com/NixOS/nixpkgs/issues?q=is%3Aissue+is%3Aopen+sort%3Areactions-%2B1-desc

--- a/pkgs/applications/misc/opencpn/default.nix
+++ b/pkgs/applications/misc/opencpn/default.nix
@@ -43,6 +43,7 @@
   util-linux,
   wxGTK32,
   xorg,
+  xz,
 }:
 
 stdenv.mkDerivation (finalAttrs: {
@@ -104,6 +105,7 @@ stdenv.mkDerivation (finalAttrs: {
     sqlite
     tinyxml
     wxGTK32
+    xz
   ]
   ++ lib.optionals stdenv.hostPlatform.isLinux [
     alsa-utils

